code-challenges.js

```diff
@@ -30,7 +54,16 @@ var myNumbers2 = [8, -7, 22, 9, 13]
 var myString1 = "bravo"
 var myString2 = "charlie"
 
+//make a function that takes a string
+const stringReverser = (word) => {
+    var splitString = word.split("") //assigns a new variable the word argument split into an array
+    var reverseString = splitString.reverse() //assigns a new variable with the letters reversed in the array
+    var joinString = reverseString.join("") //joins the reversed array of letters back into a string and removes the quotes
 
+    return joinString //returns the reversed word
+}
+//return the string with characters reversed
+console.log(stringReverser(myString2));
```

Comment:
This is great work here. I like this style of coding because it helps me clearly see what happneing at all steps. This style of code is extrmely easy to refactor down to one line if you feel compelled.